sktime/forecasting/multiplexer.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+from sktime.forecasting.base._sktime import _OptionalForecastingHorizonMixin
+from sktime.forecasting.base._sktime import _SktimeForecaster
+import copy
+
+__author__ = ["Kutay Koralturk"]
+__all__ = ["Multiplexer"]
+
+
+class Multiplexer(_OptionalForecastingHorizonMixin, _SktimeForecaster):
+    def __init__(
+        self,
+        components: dict,
```

Comment:
I'd follow scikit-learn's composition API here and make it a list of tuples in which each tuple is of the form (name, estimator), see e.g. the Pipeline in scikit-learn. 

Instead of "components", I'd call it "estimators" for the same reason as above.